libs/agno/agno/storage/session/workflow.py

```diff
@@ -59,3 +60,201 @@ def from_dict(cls, data: Mapping[str, Any]) -> Optional[WorkflowSession]:
             created_at=data.get("created_at"),
             updated_at=data.get("updated_at"),
         )
+
+
+@dataclass
+class WorkflowSessionV2:
```

Comment:
Lets rather move to a v2 file and name `WorkflowSession`

Reply:
but if we have it by the same name `WorkflowSession` it'll conflict in the `storage/sqlite.py`, between `agent/teams/workflowV1` and `workflowV2`
<img width="535" alt="image" src="https://github.com/user-attachments/assets/7290e689-c5fa-49f5-b070-95a8d0917567" />

Reply:
We can just fix it where it is imported in that file? I.e. `from agno.storage.session.workflow_v2 import WorkflowSession as WorkflowSessionV2.

Reply:
on as an alias there, yeah works. Was just confirming